src/common/Configuration.cpp

```diff
@@ -129,6 +129,7 @@ static const QHash<QString, QVariant> asmOptions = {
     { "asm.marks",          false },
     { "asm.refptr",         false },
     { "asm.flags.real",     true },
+    { "asm.reloff",         true },
```

Comment:
This should be disabled by default :)
```suggestion
    { "asm.reloff",         false },
```